apps/mosaico/src/Main.purs

```diff
@@ -142,6 +143,7 @@ notFound { params: { path} } = do
           , onLogin: pure unit
           , user: Nothing
           , article: Just notFoundArticle
+          , uuid: ""
```

Comment:
If there's a case where a value is not there, we should tend to use `Maybe`. So instead of `String` the type should be `Maybe String`.

Reply:
Changed it to a `Maybe String`